### PR TITLE
Adds `docker compose up` instantiation

### PR DESCRIPTION
**Pull request by zackees**
Missing from the README.md are instructions for launching a demo.

This pull request adds a `docker-compose.yml` which
  * exposes ports 3000, 3001 and publishes them
  *  Updates the readme to include `docker compose up` and `docker compose down`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ SmoothMQ lets you run a private SQS instance on any cloud.
 
 <!-- A drop-in replacement for SQS designed for great developer experience and efficiency. -->
 
+## Quick Start
```
Comment (poundifdef) [inline README.md:9]: There is already a `Getting Started` and `Running` section in the readme - maybe `Docker Compose` could be a subheading there.

**Comment by poundifdef** [comment]
Thanks! I'm not a big docker user myself but I'm happy to include a docker compose.

File: README.md (hunk 1 above)
Comment (zackees) [reply README.md:9]: I don't have this on a branch so edits on this are hard.

**Comment by zackees** [comment]
ah geez, looks like a bunch of changes got pulled in

File: .env
Comment (michaeltoohig) [inline .env:1]: Just checking this wasn't an accident committing your .env file

**Pull request closed without merging**